### PR TITLE
doc/cephfs/nfs: Add rook pod restart note, export and log block example

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/48914
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
see also #38913

**Review by fullerdj — APPROVED** [review]

**Comment by varshar16** [comment]
@jtlayton ping

File: doc/cephfs/fs-nfs-exports.rst [hunk 1]
```diff
@@ -113,6 +144,9 @@ Reset NFS Ganesha Configuration
 
 This removes the user defined configuration.
 
+.. note:: With rook deployment ganesha pods needs to be explicitly restarted
+   for the new config blocks to be effective.
```
Comment (jtlayton) [inline doc/cephfs/fs-nfs-exports.rst:148]: nit: "With a rook deployment, ganesha pods must be explicitly restarted for the new config blocks to be effective."

**Review by jtlayton — APPROVED** [review]
Looks fine, other than a minor grammar nit.

File: doc/cephfs/fs-nfs-exports.rst (hunk 1 above)
Comment (varshar16) [reply doc/cephfs/fs-nfs-exports.rst:148]: Done

**Pull request merged**